### ISSUE TITLE
Form validations

### ISSUE THREAD

**Issue by uday-samsani** [uday-samsani/masterpass#12]
- Forms in `/login` & `/register` need to validate forms and alert the user.
- Other forms like password, card & text should have good forms with optional fields rather than required ones.
![Screenshot from 2020-03-07 17-16-27](https://user-images.githubusercontent.com/23312297/76142834-69d33900-6097-11ea-9247-66b13bb25705.png)

**Comment by siddhantchimankar** [comment]
Hello, beginner here. Can I work on this?

**Comment by uday-samsani** [comment]
I would be happy.

**Comment by uday-samsani** [comment]
Hello @siddhantchimankar ,
I would be happy to have you develop the code. Make a new branch and start working. Glad to be on the same side of the force.

**Comment by siddhantchimankar** [comment]
Can you brief me a bit about the project and the problem exactly

**Comment by uday-samsani** [comment]
yeah sure i ll update the readme.md and can u give me ur mail address @siddhantchimankar .

**Comment by siddhantchimankar** [comment]
siddhantchimankar10@gmail.com
